### ISSUE TITLE
Win98 images are not running (stable)

### ISSUE THREAD

**Issue by henk717** [MartyShepard/Dosbox--Optionals-#3]
When using my Windows 98 SE image that works great with DOSBox-X and DOSBox Daum i either get a prompt that the machine immediately wishes to reboot, or in other attempts it does get to the desktop but instead of configuring the devices. While intended for Win9X gaming this build may miss required patches to get it to run stable.

In addition i am unable to install Windows 98 SE from an ISO like i am in DOSBox-X / DOSBox Daum because i am unable to detect the CD Drive.

**Comment by MartyShepard** [comment]
Yes, the emulated interface for the drives is missing. What is integrated into Dosbox-x and Daum. That is why the CDRom can not be found. At the moment I do not have much time to work hard at the same time and not to stray too far from the standard SVN path.

For a Win9x installation I use VMware at that time. Then I removed most device drivers and logged in to DOSBox and gladen. After that, most drivers were found by themselves. At the end is a very simplified Win9x system. The most important:
For graphics: S3 Trio64 + (765) v2.02.04
For Sound: Creative Soundblaster v4.37.00.1998

If you use the cputype = pentiumpro_slow set cpuident to 610. Otherwise, you will receive a protection error message from the config.sys.

As a replacement for the CD-Rom, I use Deamon Tools and or the wrapper _inmm. (For CD audio). The Emulated CD-Rom In the end only for the games installation.

I myself have set up almost 100 Win9x games and 90% are very stable. A few 100% played through.

Yes, there are still games relevant bugs but also system bugs. Let's see if I can do it.

Nevertheless, I am grateful for any help and thanks for the Feeback :)
